Microsoft.DotNet.UpgradeAssistant.sln

```diff
@@ -20,7 +21,7 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution
 		LICENSE.txt = LICENSE.txt
 		.github\workflows\publish.yml = .github\workflows\publish.yml
 		README.md = README.md
-		ToolsInstaller.targets = ToolsInstaller.targets
```

Comment:
I wish these would autoupdate or just display everything in a directory

Reply:
I agree